<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="4" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="197911" started-at="2017-05-05T06:11:02Z" finished-at="2017-05-05T06:14:20Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="197911" started-at="2017-05-05T06:11:02Z" finished-at="2017-05-05T06:14:20Z">
      <class name="com.qa.reve.Base.Base">
        <test-method status="PASS" signature="launchUrl()[pri:0, instance:com.qa.reve.Base.Base@573f2bb1]" name="launchUrl" is-config="true" duration-ms="9086" started-at="2017-05-05T11:40:53Z" finished-at="2017-05-05T11:41:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchUrl -->
        <test-method status="PASS" signature="verifyLogin()[pri:0, instance:com.qa.reve.Base.Base@573f2bb1]" name="verifyLogin" duration-ms="82143" started-at="2017-05-05T11:41:02Z" finished-at="2017-05-05T11:42:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLogin -->
        <test-method status="PASS" signature="verifyAddServices()[pri:3, instance:com.qa.reve.Base.Base@573f2bb1]" name="verifyAddServices" duration-ms="95920" started-at="2017-05-05T11:42:25Z" finished-at="2017-05-05T11:44:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyAddServices -->
        <test-method status="PASS" signature="verifyAddToken()[pri:4, instance:com.qa.reve.Base.Base@573f2bb1]" name="verifyAddToken" duration-ms="19821" started-at="2017-05-05T11:44:00Z" finished-at="2017-05-05T11:44:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyAddToken -->
        <test-method status="FAIL" signature="verifyAddUser()[pri:0, instance:com.qa.reve.Base.Base@573f2bb1]" name="verifyAddUser" duration-ms="4" started-at="2017-05-05T11:44:20Z" depends-on-methods="com.qa.reve.Base.Base.verifyAddServices" finished-at="2017-05-05T11:44:20Z">
          <exception class="org.openqa.selenium.UnhandledAlertException">
            <message>
              <![CDATA[unexpected alert open: {Alert text : Duplicate token name}
  (Session info: chrome=59.0.3071.36)
  (Driver info: chromedriver=2.29.461591 (62ebf098771772160f391d75e589dc567915b233),platform=Windows NT 10.0.14393 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 2 milliseconds: null
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:37:03'
System info: host: 'Ankur', ip: '192.168.1.135', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_131'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, mobileEmulationEnabled=false, networkConnectionEnabled=false, chrome={chromedriverVersion=2.29.461591 (62ebf098771772160f391d75e589dc567915b233), userDataDir=C:\Users\MOHANA~1\AppData\Local\Temp\scoped_dir10940_26196}, takesHeapSnapshot=true, pageLoadStrategy=normal, databaseEnabled=false, handlesAlerts=true, hasTouchScreen=false, version=59.0.3071.36, platform=XP, browserConnectionEnabled=false, nativeEvents=true, acceptSslCerts=true, locationContextEnabled=true, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true, unexpectedAlertBehaviour=}]
Session ID: 9c25151098477c026c7cb5078fb4d52b
*** Element info: {Using=xpath, value=.//*[@id='users']/a}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.UnhandledAlertException: unexpected alert open: {Alert text : Duplicate token name}
  (Session info: chrome=59.0.3071.36)
  (Driver info: chromedriver=2.29.461591 (62ebf098771772160f391d75e589dc567915b233),platform=Windows NT 10.0.14393 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 2 milliseconds: null
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:37:03'
System info: host: 'Ankur', ip: '192.168.1.135', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_131'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, mobileEmulationEnabled=false, networkConnectionEnabled=false, chrome={chromedriverVersion=2.29.461591 (62ebf098771772160f391d75e589dc567915b233), userDataDir=C:\Users\MOHANA~1\AppData\Local\Temp\scoped_dir10940_26196}, takesHeapSnapshot=true, pageLoadStrategy=normal, databaseEnabled=false, handlesAlerts=true, hasTouchScreen=false, version=59.0.3071.36, platform=XP, browserConnectionEnabled=false, nativeEvents=true, acceptSslCerts=true, locationContextEnabled=true, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true, unexpectedAlertBehaviour=}]
Session ID: 9c25151098477c026c7cb5078fb4d52b
*** Element info: {Using=xpath, value=.//*[@id='users']/a}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:164)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:500)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:355)
	at com.qa.reve.addusers.AddUser.clickusers(AddUser.java:37)
	at com.qa.reve.addusers.AddUser.base(AddUser.java:101)
	at com.qa.reve.Base.Base.verifyAddUser(Base.java:88)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.UnhandledAlertException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyAddUser -->
      </class> <!-- com.qa.reve.Base.Base -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
